includes/wikia/services/templates/Pagination_index.php

```diff
@@ -1,26 +1,26 @@
-<ul class="Pagination"  <? foreach($data as $key => $val ): ?> data-<?php echo $key ?>="<?php echo htmlentities($val); ?>" <? endforeach; ?> >
+<ul class="Pagination"  <? foreach($data as $key => $val ): ?> data-<?= htmlspecialchars( $key ) ?>="<?php echo htmlentities($val); ?>" <? endforeach; ?> >
```

Comment:
Nope, `Forum_board` is the only caller that passes `data` to `Paginator` controller - the values there are hardcoded.